vendor/vendor-android/ddmlib/src/main/kotlin/com/malinskiy/marathon/android/AndroidAppInstaller.kt

```diff
@@ -79,9 +80,16 @@ class AndroidAppInstaller(
         val usedStorageThresholdInPercents = androidConfiguration.usedStorageThresholdInPercents
         if (storageUsedPercentage > usedStorageThresholdInPercents) {
             logger.warn { "On ${device.serialNumber} used more than $usedStorageThresholdInPercents% of storage" }
-            androidConfiguration.cleanupDeviceScript.let {
-                logger.info { "Launching cleanup shell script `$it`" }
-                device.safeExecuteShellCommand(it).let { logger.info { it } }
+            val appsToClean = device.safeExecuteShellCommand(INSTALLED_TEST_APPS_SCRIPT).split(" ")
+            logger.info { "Removing ${appsToClean.size} apps on ${device.serialNumber}" }
+            appsToClean.forEach {
+                val result = device.safeUninstallPackage(it)
```

Comment:
This can throw exceptions, should we catch them?

Reply:
We can log it, but ignore it - anyway, we can't do anything if some of the apps won't uninstall, but we need to see the error at least.